metricbeat/module/docker/vendor/github.com/fsouza/go-dockerclient/container.go

```diff
@@ -99,6 +99,21 @@ func (p Port) Proto() string {
 	return parts[1]
 }
 
+// HealthCheck represents one check of health.
+type HealthCheck struct {
```

Comment:
HealthCheck structure in fsouza/go-dockerclient library  have been updated in october : 
[https://github.com/fsouza/go-dockerclient/commit/45c1a814bcd9b656bdb9813a7c21d0236fc301db](url)